src/scales/scale.radialLinear.js

```diff
@@ -247,11 +247,32 @@ module.exports = function(Chart) {
 		}
 	}
 
-	function adjustPointPositionForLabelHeight(angle, textSize, position) {
-		if (angle === 90 || angle === 270) {
+	function adjustPointPositionForLabelHeight(angle, textSize, position, padding) {
```

Comment:
How about breaking the padding in its components?
Something like: (not tested)
```js
	function adjustPointPositionForLabelHeight(angle, textSize, position, padding) {
		var rad;
		var paddingValue = padding ? padding : 0;
		if (angle === 90 || angle === 270) {
			position.y -= (textSize.h / 2);
		} else if (angle > 270 || angle < 90) {
			position.y -= textSize.h;
		}
		if(paddingValue) {
			rad = helpers.toRadians(angle);
			position.x += Math.cos(rad) * paddingValue;
			position.y += Math.sin(rad) * paddingValue;
		}
	}

```

Reply:
@kurkle great idea! I've implemented it according to your comment (it just needed to switch sin/cos and change one plus to minus) & also added this new option to docs ;)